TESTING.md

```diff
@@ -35,7 +35,14 @@ with pytest.raises(SomeErrorType):
 ```
 If the above function raised _SomeErrorType_, this test would pass.  Raising a different error or if nothing is raised, fails the test.  If you are not asserting an exception was raised, it should not be caught.
 
-
+* mark asynchronous  tests with `@pytest.mark.asyncio`, otherwise they will xpass
```

Comment:
Perhaps we should consider using xpass_strict = true in pytest settings?

Reply:
unfortunately it won't count as an xpass,
it will only count as an xpass (thus `xpass_struct`) if the test was marked to xfail to begin with,

Reply:
Makes sense.